dashboard/src/style.css

```
@@ -0,0 +1,421 @@
+  --warn: #f59e0b;
+  --err: #ef4444;
+  --purple: #a78bfa;
+
+  /* Text — all pass WCAG AA on --bg/#12161e backgrounds */
+  --t1: #f1f5f9; /* primary: 13.5:1 on #0a0d12 */
+  --t2: #cbd5e1; /* secondary: 9.8:1 */
+  --t3: #94a3b8; /* tertiary: 6.2:1 */
+  --t4: #7589a3; /* labels: 4.6:1 (AA minimum) */
+
+  --mono: "SF Mono", "Cascadia Code", "Fira Code", "Menlo", monospace;
+  --sans: -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, sans-serif;
```

Stylelint (17.5.0)

[error] Expected "BlinkMacSystemFont" to be "blinkmacsystemfont" (value-keyword-case)

(value-keyword-case)